src/Campaigns/Blocks/Campaign/CampaignShortcode.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Give\Campaigns\Blocks\Campaign;
+
+/**
+ * @unreleased
+ */
+class CampaignShortcode
+{
+    /**
+     * @unreleased
+     *
+     * @param array $atts
+     *
+     * @return string
+     */
+    public function renderShortcode($atts): string
+    {
+        $this->loadAssets();
+        $attributes = $this->parseAttributes($atts);
+
+        $renderFile = GIVE_PLUGIN_DIR . 'src/Campaigns/Blocks/Campaign/render.php';
+
+        if (file_exists($renderFile)) {
```

Comment:
Do we need this check? We already know this file exists.

Reply:
I suppose not. Resolved: 507964f14597b0a48f969b7b4eb6c98756c69756